src/structure.pyx

```diff
@@ -416,6 +419,102 @@ cdef class Structure:
             freesasa_structure_free(self._c_structure)
 
 
+def getStructures(fileName,
+                   options = Structure.defaultStructureArrayOptions,
+                   classifier = None, *,
+                   selection = None):
+    """
+    Create array of structures from PDB file.
+
+    Split PDB file into several structures by either by treating
+    chains separately and/or by treating each MODEL as a separate
+    structure and/or grouping chains.
+
+    Args:
+        fileName (str): The PDB file.
+        options (dict): Specification for how to read the PDB-file
+            (see :py:attr:`.Structure.defaultStructureArrayOptions` for
+            options and default value).
+        classifier: :py:class:`.Classifier` to assign atoms radii, default is used
+            if none specified.
+        selection (str): chain group selection.
```

Comment:
I guess it's not that relevant if we do the changes above, but `selection` means something  different in this library, so if we stick with this design, the variable needs a different name, e.g. `chainGroups`.

Reply:
`options` now takes a `chain-groups` option, which was also added to the `knownOptions` set in `Structure._validate_options()`